lib/index.js

```diff
@@ -118,15 +101,6 @@ function encodeArrayBuffer(packet, supportsBinary, callback) {
  */
 
 exports.encodeBase64Packet = function(packet, callback){
-
-  if (!Buffer.isBuffer(packet.data)) {
-    var buf = new Buffer(packet.data.byteLength);
-    for (var i = 0; i < buf.length; i++) {
-      buf[i] = packet.data[i];
-    }
-    packet.data = buf;
-  }
```

Comment:
Does this mean `encodeBase64Packet` can't accept ArrayBuffer anymore (event though it has a bug)?  I'm not sure if it's ok.